content/en/docs/reference/tools/_index.md

```diff
@@ -26,7 +26,7 @@ to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resou
 
 ## Helm
 
-[`Kubernetes Helm`](https://github.com/kubernetes/helm) is a tool for managing packages of pre-configured
+[`Kubernetes Helm`](https://github.com/kubernetes/helm) is a third party managed tool for managing packages of pre-configured
```

Comment:
IMO, adding "third party managed " is unnecessary.

Reply:
Helm isn't in-project now - https://github.com/kubernetes/helm has moved to https://github.com/helm/helm

Reply:
@sftim should be remove helm from this page?

Reply:
We could use a 3rd-party content warning: `{{% thirdparty-content %}}` and make this a list of relevant tools. I think it's useful to tell readers about Helm and similar tools; Kubernetes does not provide that functionality out-of-the-box.

Reply:
Hi @RA489 .
I like the idea of adding the third party content shortcode.
What do you think about the suggested changes?
Since this PR has been open for some time, I'd like to see this PR move forward or close.
Thanks!

Reply:
updated  third party content shortcode.